### PR TITLE
refactor: remove unnecessary feature attributes. 

### PR DESCRIPTION
**Pull request by iammadab**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: grovedb/Cargo.toml [hunk 1]
```diff
@@ -7,21 +7,21 @@ edition = "2021"
 license = "MIT"
 
 [dependencies]
-rs_merkle = "1.1.0"
+rs_merkle = { version = "1.1.0", optional = true }
```
Comment (fominok) [inline grovedb/Cargo.toml:10]: Do we use rs-merkle btw? As I remember it was for a root tree and we changed that to Merk long ago

File: grovedb/Cargo.toml [hunk 2]
```diff
@@ -34,6 +34,29 @@ name = "insertion_benchmark"
 harness = false
 
 [features]
-default = ["full"]
-full = ["merk/full"]
-verify = ["merk/verify"]
\ No newline at end of file
+default = ["verify"]
```
Comment (fominok) [inline grovedb/Cargo.toml:37]: I'm wondering what will it be without any features enabled lol

It's the reason why `#[cfg(any(feature = "full", feature = "verify"))]` is required, could be just unconditional compilation, like "basic GroveDB package"

**Review by fominok — APPROVED** [review]

File: grovedb/Cargo.toml (hunk 1 above)
Comment (iammadab) [reply grovedb/Cargo.toml:10]: Yeah, true think it was used in a test, will get rid of it.

Comment (iammadab) [reply grovedb/Cargo.toml]: Done

File: grovedb/Cargo.toml (hunk 2 above)
Comment (iammadab) [reply grovedb/Cargo.toml]: Doing this in a new pr

**Review by QuantumExplorer — APPROVED** [review]

**Pull request merged**